### PR TITLE
Fix setting dropdown bug

### LINKED ISSUE #387: Setting dropdown not working
Please see the rightmost settings icon.
image

### PR DESCRIPTION
**Pull request by cesswairimu**
Fixes #387 

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by cesswairimu** [comment]
Hello @jywarren, do we have a test prod env like unstable.publiclab.org for plots2? This was working fine locally need to test remotely to make sure its working fine. Thanks

**Comment by jywarren** [comment]
Yes we do now -- @icarito just finished building it and it will be
documented shortly! Thank you!

**Comment by cesswairimu** [comment]
Actually my bad..wass able to replicate this locally and just updated this to show the "setting icon" only if a user can delete the map. Thanks

**Comment by avsingh999** [comment]
Hi @cesswairimu please solve conflicts.
thanks : )

**Comment by jywarren** [comment]
Oh, just a note that it seems @avsingh999 also submitted https://github.com/publiclab/mapknitter/pull/432 for this, thank you! I'm glad you found one another. We'll try to coordinate a bit better to avoid double-solving in the future, and thanks to all! 👍

File: app/views/maps/show.html.erb [hunk 1]
```diff
@@ -1,4 +1,14 @@
 <% content_for :title do %>
+  <div class="pull-right dropdown">
```
Comment (jywarren) [inline app/views/maps/show.html.erb]: Hmm, was this missing? Or was it perhaps to be added in via JavaScript? Sorry, i'm a bit out of sync on this. Thank you!

**Comment by cesswairimu** [comment]
@avsingh999 Actually @stefannibrasil did away with the setting icon and added this awesome layout that works fine and the issue is not there anymore..closing this and the issue. Thanks

**Comment by cesswairimu** [comment]
@avsingh999 Actually @stefannibrasil did away with the setting icon and added this awesome layout that works fine and the issue is not there anymore..closing this and the issue. Thanks

**Pull request closed without merging**